projects/acs-engine/objects.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: List
+items:
+
+# Group will have some elevated permissions to deploy/maintain cluster-operator:
+- apiVersion: user.openshift.io/v1
+  kind: Group
+  metadata:
+    name: azure-team
+  users:
+  - mjudeikis
+  - jim-minter
+  - kargakis
+  - pweil-
+  - kwoodson
+
+- apiVersion: v1
+  kind: RoleBinding
+  metadata:
+    name: admin
+    namespace: azure
+  groupNames:
+  - aos-azure
+  roleRef:
+    name: admin
+  subjects:
+  - kind: Group
+    name: aos-azure
+
+- kind: ImageStream
```

Comment:
Add this back with the builds and keep the policy resources here.

Reply:
If we create separate builds using the same template which produces different tag to same imageStream - it does not work.

Reply:
I think I am in favor of removing the templatization from the other template and keep explicit build config. In my experience it's much better to manage than an openshift template albeit more config.